File: WondayWall.iOS/WondayWall/Models/GoogleAiServiceTier.swift
import Foundation

// Gemini API の推論階層
// Flex: 標準より 50% 安価だが、レイテンシが変動しベスト・エフォート型
// Standard: 通常料金、低レイテンシ・高信頼性
enum GoogleAiServiceTier: String {
    case standard
    case flex
}

File: WondayWall.Android/app/src/main/kotlin/com/studiofreesia/wondaywall/models/GoogleAiServiceTier.kt
package com.studiofreesia.wondaywall.models

import kotlinx.serialization.Serializable

// Google AI サービスティア（Windows版・iOS版と同名で揃える）
@Serializable
enum class GoogleAiServiceTier {
    // 通常の推論（手動生成時はこちらを使う）
    Standard,
    // コスト効率重視のバックグラウンド推論（容量不足時は Standard にフォールバック）
    Flex,
}
